src/Microsoft.Azure.Relay/HybridConnectionClient.cs

```diff
@@ -178,16 +181,6 @@ public HybridConnectionClient(string connectionString, string path)
                     await webSocket.ConnectAsync(webSocketUri, cancelSource.Token).ConfigureAwait(false);
                 }
 
-#if NET45 // TODO: Flow Response Headers in NETSTANDARD ClientWebSocket
-                var trackingId = webSocket.ResponseHeaders[TrackingContext.TrackingIdName];
-                if (!string.IsNullOrEmpty(trackingId))
-                {
-                    // Update to the flown trackingId (which has _GX suffix)
-                    trackingContext = TrackingContext.Create(trackingId, trackingContext.SubsystemId);
-                    traceSource = nameof(HybridConnectionClient) + "(" + trackingContext.ToString() + ")";
-                }
-#endif
```

Comment:
This code was merely grabbing the returned "TrackingId" HTTP header in order to have a tracking ID with _GXX suffix.